src/b3d/chisight/gen3d/pixel_kernels/pixel_rgbd_kernels.py

```diff
@@ -27,69 +27,31 @@ def is_unexplained(latent_value: FloatArray) -> bool:
 
 
 @Pytree.dataclass
-class PixelRGBDDistribution(genjax.ExactDensity):
-    """
-    Distribution args:
-    - latent_rgbd: 4-array: RGBD value.  (a value of [-1, -1, -1, -1] indicates no point hits here.)
-    - color_scale: float
-    - depth_scale: float
-    - visibility_prob: float
-    - depth_nonreturn_prob: float
-    - intrinsics: dict
-    - depth_nonreturn_prob_for_invisible: float
-        Depth nonreturn prob for pixels not associated with a latent point,
-        or for "invisible" pixels.
-
-    The support of the distribution is [0, 1]^3 x ([near, far] + {DEPTH_NONRETURN_VALUE}).
-
-    Note that this distribution expects the observed_rgbd to be valid. If an invalid
-    pixel is observed, the logpdf will return -inf.
+class PixelDistFromColorDistAndDepthReturnDist(genjax.ExactDensity):
```

Comment:
I renamed what I had called "RGBDDist" to this more descriptive name, to hopefully clarify the role this combinator plays.